packages/@react-spectrum/button/src/Button.tsx

```diff
@@ -65,15 +109,17 @@ function Button<T extends ElementType = 'button'>(props: SpectrumButtonProps<T>,
         data-variant={variant}
         data-style={style}
         data-static-color={staticColor || undefined}
+        aria-disabled={isPending}
```

Comment:
```suggestion
        aria-disabled={isPending || undefined}
```
just so that false doesn't interfere with the aria-disabled that might come from `useButton`'s returned properties
https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/button/src/useButton.ts#L82

ideally, this would be set in `useButton` as well, trying to decide if this is a Spectrum specific thing. it might coincide with 'focusable but disabled' https://github.com/adobe/react-spectrum/issues/3662